ansible/roles/helm/tasks/main.yml

```diff
@@ -21,3 +21,6 @@
   when: helm_version is failed
   run_once: true
 
+- name: Wait for Tiller to become responsive.
+  wait_for:
+    timeout: 120
```

Comment:
We could reduce this. Will do a couple of runs to know what is a optimistic sleep time.

Reply:
How about we actually put in place a poll or push notification? I believe `kubectl` will work for this:

```
kubectl rollout status --watch [deployment/service-name] --namespace=[namespace]
```

Reply:
Cool, that should work ! Thanks